src/main/drivers/serial_uart_pinconfig.c

```diff
@@ -46,6 +46,12 @@ void uartPinConfigure(const serialPinConfig_t *pSerialPinConfig)
     for (size_t hindex = 0; hindex < UARTDEV_COUNT_MAX; hindex++) {
```

Comment:
> I would still prefer defining all USE_USARTx that can be user-configured

Yes. The proposed `uartPinConfigure` is written that way.

> eventually removing it.

I don't know about this. We may start with explicitly defining all possible USE_UARTx in target.h, but without pin spec. I think we should review everything when we default to this anyways.